src/main/java/com/google/sps/data/SampleData.java

```diff
@@ -177,19 +177,27 @@ public void addAvailabilityByTutorEmail(String email, TimeRange timeslot) {
     /** Adds the given TutorSession to the scheduled sessions array of the given tutor. */
     public void addToTutorScheduledSessionsByEmail(String email, TutorSession tutoringSession) {
         Tutor tutor = getTutorByEmail(email);
-
-        // tutors.remove(tutor);
         tutor.addToScheduledSessions(tutoringSession);
-        // tutors.add(tutor);
+    }
```

Comment:
Just a heads up, I don't think we'll be needing the mock datastore services anymore since I'm using the local datastore helper to test now.

Reply:
Oh sounds good! I won't use them for other servlets I make then